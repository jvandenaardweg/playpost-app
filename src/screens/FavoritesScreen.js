import React from 'react';
import { EmptyState } from '@/components/EmptyState';

export default class FavoritesScreen extends React.Component {
  static navigationOptions = {
    title: 'Favorites'
  };

  render() {
    return (
      <EmptyState title="Your favorite articles" description="Articles you really liked can be added to your favorites" />
    );
  }
}
