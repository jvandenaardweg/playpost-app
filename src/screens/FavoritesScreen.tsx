import React from 'react';

import { AppBackground } from '../components/AppBackground';
import { InteractionManaged } from '../components/InteractionManaged';
import { PlaylistContainer } from '../containers/PlaylistContainer';

export class FavoritesScreen extends React.PureComponent {
  render() {
    return (
      <AppBackground>
        <InteractionManaged showActivityIndicator>
          <PlaylistContainer isFavoriteScreen />
        </InteractionManaged>
      </AppBackground>
    );
  }
}
