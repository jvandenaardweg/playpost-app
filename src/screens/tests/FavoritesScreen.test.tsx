import React from 'react';
// tslint:disable-next-line: no-submodule-imports
import * as ShallowRenderer from 'react-test-renderer/shallow';

import { FavoritesScreen } from '../FavoritesScreen';

describe('FavoritesScreen', () => {

  describe('rendering', () => {
    let wrapper: any;

    beforeEach(() => {
      const renderer = ShallowRenderer.createRenderer();
      renderer.render(<FavoritesScreen />);
      wrapper = renderer.getRenderOutput();
    });

    it('should render correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
