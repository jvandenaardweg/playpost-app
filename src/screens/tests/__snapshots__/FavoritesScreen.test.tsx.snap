// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesScreen rendering should render correctly 1`] = `
<Memo>
  <Memo
    showActivityIndicator={true}
  >
    <Memo(Connect(Component))
      isFavoriteScreen={true}
    />
  </Memo>
</Memo>
`;
